Makefile

```diff
@@ -96,7 +97,13 @@ lint: $(GOLANGCI_LINT)
 
 fmt:
 	@echo gofmt		# Show progress, real gofmt line is too long
-	find pkg cmd test -name '*.go' | xargs -r gofmt -s -l -w
+	@if [ $(UNAME_S) == 'Linux' ] ; then \
+		find pkg cmd test -name '*.go' | xargs -r gofmt -s -l -w; \
+	fi
+	#BSD (MacOS) xargs does not have option -r (--no-run-if-empty)
+	@if [ $(UNAME_S) = 'Darwin' ] ; then \
+		find pkg cmd test -name '*.go' | xargs gofmt -s -l -w; \ #BSD xargs does not have this behavior, so there's no need to disable it.
```

Comment:
If the whole fix is about `xargs`, could you check if you could use the one from brew?

Reply:
I think is nothing to fix in `xargs` side, just different behavior

Reply:
If you use the xargs from homebrew aka from GNU, you should get the same behavior right?